http/http_client.hpp

```diff
@@ -529,10 +597,30 @@ class HttpClient : public std::enable_shared_from_this<HttpClient>
     }
     void sendData(const std::string& data)
     {
+        auto eventData = nlohmann::json::parse(data);
```

Comment:
I think you need to turn off the exceptions 
parse(data, nullptr, false)

Reply:
Done